### PR TITLE
CPR-498 Add console appenders

### PR DESCRIPTION
**Pull request by H-Iwanejko**
* Add console appenders, based of template repository. 
* As no console logs are written currently so not visible during (kubectl logs)

### CODE REVIEW

File: src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -6,7 +6,33 @@
...
+
+    <logger name="org.springframework" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="com.microsoft.applicationinsights" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="org.apache.catalina" additivity="false" level="INFO">
```
Comment (markreesmoj) [inline src/main/resources/logback-spring.xml:31]: I would go with WARN for most of these - do we really care?

File: src/main/resources/logback-spring.xml [hunk 2]
```diff
@@ -6,7 +6,33 @@
...
     <springProperty scope="context" name="app" source="spring.application.name"/>
 
-    <logger name="uk.gov.justice.digital.hmpps.personrecord" additivity="false" level="DEBUG">
+    <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>${LOG_PATTERN}</pattern>
+        </encoder>
+    </appender>
+
+    <logger name="uk.gov.justice.digital.hmpps.personrecord.HmppsPersonRecordKt" additivity="false" level="DEBUG">
```
Comment (markreesmoj) [inline src/main/resources/logback-spring.xml:15]: specified on line 19 I think

File: src/main/resources/logback-spring.xml [hunk 3]
```diff
@@ -6,7 +6,33 @@
 
     <springProperty scope="context" name="app" source="spring.application.name"/>
 
-    <logger name="uk.gov.justice.digital.hmpps.personrecord" additivity="false" level="DEBUG">
+    <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>${LOG_PATTERN}</pattern>
+        </encoder>
+    </appender>
```
Comment (markreesmoj) [inline src/main/resources/logback-spring.xml:13]: :-)
DIdn't notice we didn't have one!

Good work tidying this area up anyway

**Review by markreesmoj — COMMENTED** [review]
💬

File: src/main/resources/logback-spring.xml [hunk 4]
```diff
@@ -6,7 +6,33 @@
...
+
+    <logger name="com.microsoft.applicationinsights" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="org.apache.catalina" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="org.springframework.boot" additivity="false" level="INFO">
```
[the remark this answers is not in the document]
Comment (H-Iwanejko) [reply src/main/resources/logback-spring.xml]: Yeah reduced the scope + dup of the logback

File: src/main/resources/logback-spring.xml (hunk 2 above)
Comment (H-Iwanejko) [reply src/main/resources/logback-spring.xml]: Removed 👍

File: src/main/resources/logback-spring.xml (hunk 1 above)
Comment (H-Iwanejko) [reply src/main/resources/logback-spring.xml]: Just set to INFO for all of them as most of the noisy logs are set to DEBUG

**Review by markreesmoj — APPROVED** [review]
👍

**Pull request merged**